srcdeps-maven-quickstarts/srcdeps-mvn-git-revision-non-master-quickstart/.mvn/extensions.xml

```diff
@@ -23,4 +23,9 @@
     <artifactId>srcdeps-maven-local-repository</artifactId>
     <version>2.0.2</version><!-- @srcdeps.version@ -->
   </extension>
+  <extension>
+    <groupId>org.srcdeps.mvn</groupId>
+    <artifactId>srcdeps-maven-enforcer</artifactId>
+    <version>2.0.2</version><!-- @srcdeps.version@ -->
```

Comment:
Should it refer to the previous version? Or should it refer to "current" (2.0.3-SNAPSHOT)?